src/Resources/config/services.xml

```diff
@@ -177,5 +183,17 @@
         <service id="League\Bundle\OAuth2ServerBundle\Security\Authentication\Token\OAuth2TokenFactory">
             <argument type="string" />
         </service>
+
+        <!-- PSR-7/17 -->
+        <service id="nyholm.psr7.psr17_factory" class="Nyholm\Psr7\Factory\Psr17Factory" />
+
+        <service id="http_message_factory" class="Symfony\Bridge\PsrHttpMessage\Factory\PsrHttpFactory">
+            <argument type="service" id="nyholm.psr7.psr17_factory" />
+            <argument type="service" id="nyholm.psr7.psr17_factory" />
+            <argument type="service" id="nyholm.psr7.psr17_factory" />
+            <argument type="service" id="nyholm.psr7.psr17_factory" />
+        </service>
+
+        <service id="http_foundation_factory" class="Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory" />
```

Comment:
I'm not sure about these namings. I'll take any advice

Reply:
let's prefix them by `league.oauth2_server.`